tests/contrib/actions/slack/test_slack_message_manager.py

```diff
@@ -1,5 +1,17 @@
 import pytest
-from spidermon.contrib.actions.slack import SlackMessageManager
+
+from scrapy.crawler import CrawlerRunner
+from scrapy.spiders import Spider
+
+from spidermon.contrib.actions.slack import (
+    SlackMessageManager,
+    SendSlackMessage,
+)
+
+
+def get_crawler(settings=None):
```

Comment:
You can use `scrapy.utils.test.get_crawler` function available in Scrapy (which is pretty much the same you did), so we can avoid problems if Scrapy changes the way a test instance of crawler is done.